packages/memory-inspector/src/browser/style/index.css

```
@@ -394,8 +394,11 @@ table.t-mv-view .t-mv-view-address {
 
 .mw-input-select .theia-input {
   position: absolute;
+  top: 1px;
+  left: 1px;
   z-index: 1;
   width: calc(100% - 18px);
+  padding-block: unset;
 }
```

```suggestion
.mw-input-select .theia-input {
  position: absolute;
  z-index: 1;
  width: calc(100% - 18px);
  border: var(--theia-border-width) solid var(--theia-dropdown-border);
  line-height: calc(var(--theia-content-line-height) - 2px);
}
```

Instead of adding a `top` and `left` offset, wouldn't the solution above be more clean?